components/collection/CollectionInfo.vue

```diff
@@ -29,13 +29,17 @@
               <div class="bg-background-color border py-2 px-4 text-xs">
                 Recipient Address:
                 <nuxt-link
+                  v-if="isValidAddress(recipient)"
```

Comment:
but valid address is also evm address on Base

Reply:
then what should it link to ?

Reply:
I am just telling that valid address depends on chain prefix.

Reply:
What solution do you suggest?